smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptSettings.java

```diff
@@ -295,6 +299,19 @@ public void setDisableDefaultValidation(boolean disableDefaultValidation) {
         this.disableDefaultValidation = disableDefaultValidation;
     }
 
+    /**
+     * Returns whether or not required members are backward-compatible.
+     *
+     * @return true if required members are backward-compatible. Default: true
+     */
+    public boolean isBackwardCompatibleRequiredMember() {
```

Comment:
The name, `BackwardCompatibleRequiredMember`, isn't descriptive of what this is actually doing. Is there another suggestion that will make this more obvious when it shows up in smithy-build.json?

Reply:
what about calling it: `backwardCompatibilityMode`?

Reply:
That makes more sense. Maybe `memberNullabilityCompatibilityMode`?